packages/ERTP/src/issuer.js

```diff
@@ -23,7 +23,7 @@ function makeIssuerKit(
   displayInfo = undefined,
 ) {
   assert.typeof(allegedName, 'string');
-  assertDisplayInfo(displayInfo);
+  displayInfo = coerceDisplayInfo(displayInfo);
```

Comment:
Good you changed the assert into a coerce and let it do the coercion. Better than what I suggested.